tests/sdk/v2/driver/resp_mocks.py

```diff
@@ -276,6 +276,17 @@ def make_get_wf_run_logs_response():
     return "bytes"
 
 
+def make_get_wf_run_system_logs_response():
+    return "bytes"
+
+
+def make_get_wf_run_system_logs_response_with_content():
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏Can you follow the pattern that Alex mentioned on the call we had? By making this a file that is read in